pkg/kyverno/common/fetch.go

```diff
@@ -3,8 +3,11 @@ package common
 import (
 	"encoding/json"
 	"errors"
+	"fmt"
 	"io/ioutil"
 
+	"github.com/kyverno/kyverno/pkg/kyverno/sanitizedError"
+
 	v1 "github.com/kyverno/kyverno/pkg/api/kyverno/v1"
 	"github.com/kyverno/kyverno/pkg/client/clientset/versioned/scheme"
```

Comment:
@NoSkillGirl The scheme not found issue was due to this incorrect import, let's change `"github.com/kyverno/kyverno/pkg/client/clientset/versioned/scheme"` to `"k8s.io/client-go/kubernetes/scheme"`. 
Then you should be able to remove registers from `/pkg/client/clientset/versioned/scheme/register.go`.